app/services/force/event_subscriber_translate_service.rb

```diff
@@ -35,15 +35,19 @@ def listen_and_process_events
         # available methods for the subscription instance:
         #   https://www.rubydoc.info/github/eventmachine/eventmachine/EventMachine/Deferrable
         #   https://www.rubydoc.info/gems/faye/Faye/Subscription
-        subscription = subscribe_to_listing_updates
-        subscription.callback do
-          Rails.logger.info('Subscribed to Salesforce Platform Events')
-        end
-        subscription.errback do |error|
-          logger(
-            "Error subscribing to Salesforce Platform Events: #{error.inspect}",
-            error: true,
-          )
+        if Unleash::Client.new.is_enabled? 'GoogleCloudTranslate'
```

Comment:
question: Why put the feature flag here, as opposed to the top of the `listen_and_process_events` method, the `initialize` method, or the `translate.rake` file?

Reply:
It needed to be inside the loop so it can catch the case where we turn the feature flag off after it's already been running